extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -32,13 +32,18 @@ export class WizardController {
 		const wizard = azdata.window.createWizard(WIZARD_TITLE, 'wide');
 		wizard.generateScriptButton.enabled = false;
 		wizard.generateScriptButton.hidden = true;
-
-		const sourceConfigurationPage = new SourceConfigurationPage(wizard, stateModel);
-		const skuRecommendationPage = new SKURecommendationPage(wizard, stateModel);
-		const subscriptionSelectionPage = new SubscriptionSelectionPage(wizard, stateModel);
+		// Disabling unused pages
+		// const sourceConfigurationPage = new SourceConfigurationPage(wizard, stateModel);
+		// const skuRecommendationPage = new SKURecommendationPage(wizard, stateModel);
```

Comment:
The sourceConfigurationPage and the skuRecommendationPage are the ones I am working on, can we keep those?

Reply:
I see, Won't it come after azure accounts page?

Reply:
yeah it will, I guess as long as we are planning to uncomment them later it is fine and I will just keep them uncommented in my branch